edge-modules/load-gen/Settings.cs

```diff
@@ -29,20 +29,31 @@ public class Settings
                     configuration.GetValue<ulong>("messageSizeInBytes", 1024),
                     configuration.GetValue("transportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue("outputName", "output1"),
-                    configuration.GetValue("startDelay", TimeSpan.FromSeconds(2)));
+                    configuration.GetValue("testStartDelay", TimeSpan.FromMinutes(5)),
+                    configuration.GetValue("testRunDuration", TimeSpan.MaxValue),
+                    configuration.GetValue("trackingId", string.Empty));
             });
 
         Settings(
```

Comment:
Lee made a comment about DefaultSettings not needing Lazy. Do you agree?

Reply:
To answer your question, I would say both are ok; and it is not so critical.  For long answer, please read this post, https://csharpindepth.com/articles/singleton.

Reply:
If you want to simplify it without using Lazy, please help to create another PR to update all Settings class in Edge solution.  thanks.